### ISSUE TITLE
Suggestion: allow cookie-based time offsetting

### ISSUE THREAD

**Issue by fbordas** [Mitsunee/fgo-time#4]
Some countries around the world, especially in Caribbean zones, have odd timezones that sometimes don't really match with their auto-detected geolocation. It'd be useful to allow (power?) users to adjust the displayed time with a manual timezone offset setting so the times shown match with their actual time. (Personal case: I'm located in Dominican Republic, displayed time shows 2am when current time is actually 1am at the time of creating this issue.)

**Comment by Mitsunee** [comment]
I am using the local system time for the local clock, so it should be correct unless your system isn't reporting its time properly. I also don't see what cookies what do here other than making me have to follow EU Regulations across all of my projects that don't use cookies.

**Comment by fbordas** [comment]
Odd, because as shown in the attached image, the displayed time and the system time don't match.

![image](https://user-images.githubusercontent.com/12531448/89182734-98810d00-d56c-11ea-9fa0-63d2bc17c6b1.png)

**Comment by Mitsunee** [comment]
can you open the dev console (F12 or CTRL+Shift+i), run the following command and screenshot that alongside your taskbar's clock for comparison?
```js
let debugDate = new Date, debugSpaceTime = spacetime.now();
clear();
console.log("localToUTC", localToUTC);
console.log("timeUTC", debugDate.toUTCString());
console.log("timeLocal", debugDate.toLocaleTimeString());
console.log("timeNow", debugSpaceTime.toLocalDate());
console.log("timeFormatted", debugSpaceTime.format(clockFormat).toUpperCase());
```

**Comment by fbordas** [comment]
I'm pretty sure this is due to some weird timezone offsetting. Console is reporting one hour faster than my actual time.

![image](https://user-images.githubusercontent.com/12531448/89187228-61622a00-d573-11ea-8603-31e550cf98fb.png)

**Comment by Mitsunee** [comment]
just to verify, does http://www.time-time.net/timer/digital-clock.php also show the same incorrect time and also in other browsers (Edge or Firefox for example)? I've done some digging and not found anything about this issue.

**Comment by fbordas** [comment]
Yes, other browsers are reporting the same incorrect time. Console reports browser is getting "America/Halifax" as timezone. Weird, because system options has it set to Georgetown/La Paz/Manaus/San Juan (UTC-4). No clue what's causing the incorrect timezone detection.

I'll leave this open to hear your input, but since this is evidently a client-side issue, feel free to close this. (Though I still stand by the idea of having a time offset picker, heh.)

**Comment by Mitsunee** [comment]
I had another person test that same timezone settings on windows 10 and they're getting "America/La_Paz" as their timezone with the js function above. Could you try changing your timezone to something else and then back to see if that convinces your browser(s)? Current theory is that windows derped somewhere and changing the setting might fix it. You'll probably want to close the browser before changing your timezone just in case.

**Comment by fbordas** [comment]
I closed this and forgot to post an explanatory comment.

I changed my timezone and restarted, and apparently that was enough to correct the time offsetting. Console now correctly reports "America/Santo_Domingo" as current timezone. Windows is dumb and when choosing to select timezone automatically it defaults to "America/Halifax" for some reason. I set the timezone manually now and no longer have any discrepancies.
